crates/core/tedge/src/cli/init.rs

```diff
@@ -127,6 +127,14 @@ impl TEdgeInitCmd {
                 Some(0o775),
             ),
         )?;
+        create_directory(
+            config_dir.join("operations").join("c8y"),
```

Comment:
We may have to undo this in future as the desire is to make `tedge` and other mappers as independent as possible. The right thing to do would have been to make the `tedge-mapper-c8y` create this directory as part of its installation as well, in addition to runtime, so that plugins like `c8y-remote-access` plugin, can be sure that this directory is present, before it adds its own operation file. For that to work, such plugins should have a `Dependends` relationship defined on the `tedge-mapper-c8y` package, but that's not an unreasonable requirement.

But, for the current state of things, this quick fix is seems sufficient.